<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-8GPJC4Q" failures="1" tests="1" name="Default test(failed)" time="0.106" errors="0" timestamp="2022-11-07T19:58:45 IST">
  <testcase classname="Assserionspack.failtestclass" name="TC02" time="0.017">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Assserionspack.failtestclass.TC02(failtestclass.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC02 -->
  <testcase classname="Assserionspack.failtestclass" name="TC01" time="0.0">
    <ignored/>
  </testcase> <!-- TC01 -->
  <testcase classname="Assserionspack.failtestclass" name="TC03" time="0.0">
    <ignored/>
  </testcase> <!-- TC03 -->
</testsuite> <!-- Default test(failed) -->
